sqlmesh/utils/errors.py

```diff
@@ -119,6 +119,11 @@ class UnsupportedCatalogOperationError(EngineAdapterError):
     pass
 
 
+class CircuitBreakerError(SQLMeshError):
+    def __init__(self) -> None:
+        super().__init__("Circuit breaker has been triggered.")
```

Comment:
this error is a bit cryptic

Reply:
It should never be user-facing though. The idea is that it's an internal mechanics and someone's always expected to intercept it and provide a friendlier explanation.